packages/adk/lib/util/space.ts

```diff
@@ -0,0 +1,19 @@
+import fs from 'fs';
+export const ACTION_CONFIG_FILE = '.actionconfig';
+
+export function read_space(): string {
```

Comment:
Instead of doing it this way, I suggest to inject this modeled information to generators.

Reply:
Good call. I will track it separately, as a refactoring AI, if you don't mind. This change is not required right now as it won't affect the generated readme.